### DISCUSSION TITLE
Dart scss no longer supports import

### DISCUSSION THREAD

**Discussion by manuelkleinert** [uikit/uikit#5076] [General]
Hello together
I have the problem that import will soon no longer be supported with the new Dart Sass. Uikit still has a lot of imports in their scss. Will there be a solution soon?

Thanks for your answer

https://sass-lang.com/blog/import-is-deprecated/
https://github.com/sass/sass/issues/3513

**Comment by Ulrich-Mbouna** [comment]
Same Problem

**Comment by sonhoang89** [comment]
I am also interested in this issue and would like the 'sass' update to switch from 'Import' to '@use'

**Comment by eshimischi** [comment]
Just use previous version of SASS, what is the issue here indeed? Transition to the latest SASS standards isn't a 1 day build up, give devs a proper time!

**Comment by 67a2** [comment]
This! Anyone know if updated Sass files are on the horizon?

**Comment by janschoenherr** [comment]
We are currently looking into it. 🙋‍♂️
